digdag-core/src/main/java/io/digdag/core/agent/OperatorManager.java

```diff
@@ -234,17 +236,27 @@ private void runWithWorkspace(Path projectPath, TaskRequest request)
             shouldBeUsedKeys.remove(operatorKey.get());
         }
 
-        CheckedConfig checkedConfig = new CheckedConfig(config);
+        // Now config is evaluated using JavaScript. Copy values from it
+        // to create evaluated localConfig (original localConfig is not
+        // evaluated).
+
+        Config localConfig = config.getFactory().create();
+        for (String localKey : request.getLocalConfig().getKeys()) {
+            localConfig.set(localKey, config.get(localKey, JsonNode.class).deepCopy());
```

Comment:
What if `localConfig` has `schema: myschema` and `config` has `schema: {"foo": "bar"}`? If I understand correctly, it seems an unexpected value in `config` can be copied to `localConfig`.

How about evaluating `localConfig` using `evalEngine`?